streams/src/test/java/org/apache/kafka/streams/integration/GlobalThreadShutDownOrderTest.java

```diff
@@ -74,7 +75,7 @@
     private KafkaStreams kafkaStreams;
     private String globalStoreTopic;
     private String streamTopic;
-    private final List<Long> retrievedValuesList = new ArrayList<>();
+    private final Set<Long> retrievedValuesList = new HashSet<>();
```

Comment:
nit: we should update the name too if we're changing this from a list

Reply:
ack